docs/plans/2026-03-31-weixin-qqbot-channel-bridge-support-design.md

```
@@ -73,7 +73,7 @@ conclusion is:
 
 **Chosen Design**
 
-## 1. Add a plugin-backed support tier to the channel model
+### 1. Add a plugin-backed support tier to the channel model
```

markdownlint-cli2 (0.22.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)